scripts/pkg/build_templates/opensearch-dashboards/deb/debian/postinst

```diff
@@ -35,6 +35,10 @@ echo " sudo systemctl daemon-reload"
 echo " sudo systemctl enable opensearch-dashboards.service"
 echo "### You can start opensearch-dashboards service by executing"
 echo " sudo systemctl start opensearch-dashboards.service"
+echo "### Upcoming breaking change in packaging"
+echo " We are planing to tighten the permissions of installed files in a future version of the opensearch-dashboards package."
+echo " If you setup tooling that needs read access to the OpenSearch Dashboards configuration files, we invite you to add the user running these tools to the 'opensearch-dashboards' group."
```

Comment:
`your` not `you`?

Reply:
I feel like it's good, but maybe "If you are configuring tools that require read access..." is simpler and better?